app/views/sign_up/completions/show.html.erb

```diff
@@ -24,6 +24,18 @@
         sp: content_tag(:strong, decorated_session.sp_name),
       ) %>
 </p>
+<% if @multiple_factors_enabled %>
+  <%= render(AlertComponent.new(type: :warning, class: 'margin-bottom-4')) do %>
+    <%= link_to(
+          t('multi_factor_authentication.second_method_warning.link'),
+          login_two_factor_options_url,
+        ) %>
+    <%= t(
+          'multi_factor_authentication.second_method_warning.text',
+        )
+  %>
```

Comment:
Nit: Can we bring the closing tag onto the same line as the closing parenthesis, for consistency with line 32?

```suggestion
        ) %>
```